go/jws/jws_header.go

```diff
@@ -32,6 +32,7 @@ func NewJwsHeaderV2(kid string, headers *orderedmap.OrderedMap) JwsHeader {
 		Kid:       kid,
 		TlVersion: "2",
 		TlHeaders: headerKeys,
+		Jku:       jku,
```

Comment:
I suppose the minor downside is we'll start inflating the regular signature jwt headers with `,"jku":""`. Can we also avoid having it when empty?